docs/changes.rst

```diff
@@ -25,8 +25,12 @@ Released: not yet
   b) incompatibility between click 8.0 and clicl-repl.
   (see issues #816 and #817)
 
-* Limit mock package to lt 4.0.3 to avoid issue issue that causes test failure.
-  (see #822)
+* Fix issue with failure in mock package version 4.0.3 by creating local
+  warnings.warn ... functions to implement our own functions that call the
+  Python warnings functions and  to remove the use of the patch decorator
+  before the definition of the cli function.  Note this superceeds previous
+  change that limited versions of mock package. (see issue #822)
+
```

Comment:
I'm not happy with this change log entry:
* The first sentence is overly complicated ("by creating local warnings.warn ... functions to implement our own functions that call the Python warnings functions").
* This change log replaces the prior change log about #822, so it does not make sense for the reader to reference the now deleted previous change log entry.

Reply:
Fixed.